lexos/interfaces/similarity_query_interface.py

```diff
@@ -48,15 +47,8 @@ def similarity():
     if 'gen-sims' in request.form:
         # 'POST' request occur when html form is submitted
         # (i.e. 'Get Graphs', 'Download...')
-        score_name_data_frame = utility.generate_similarities(file_manager)
-
-        docs_score = np.concatenate(score_name_data_frame.values)
-        docs_name = np.array(score_name_data_frame.index)
-
-        docs_list_score = '***'.join(str(score) for score in docs_score) \
-                          + '***'
-        docs_list_name = '***'.join(name for name in docs_name) + '***'
-
+        docs_list_score, docs_list_name = utility.generate_similarities(
```

Comment:
But This part cannot be merged, because that will mess up the code in master.

I will think of a way to fix this.